internal/cloudapi/openapi.yml

```diff
@@ -203,6 +204,11 @@ components:
         snapshot_name:
           type: string
           example: 'my-snapshot'
+        share_with_accounts:
+          type: array
+          example: ['123456789012']
+          items:
+            type: string
```

Comment:
Should this be numbers rather than strings?

Reply:
Hm for a number we'd have to convert it back to a string to call the aws api. And who knows we might wanna accept account aliases or something at some point?

Reply:
Fair enough, if this is the API there is no reason for us to artificially restrict it.